addons/website_slides/static/src/js/slides_course_slides_list.js

```diff
@@ -43,25 +43,6 @@ publicWidget.registry.websiteSlidesCourseSlidesList = publicWidget.Widget.extend
             placeholder: 'o_wslides_slides_list_slide_hilight position-relative mb-1'
         });
     },
-
-    /**
-     * This method will check that a section is empty/not empty
-     * when the slides are reordered and show/hide the
-     * "Empty category" placeholder.
-     *
-     * @private
-     */
-    _checkForEmptySections: function (){
```

Comment:
Can't remove methods in stable.
Is there a way to make this retro-compatible?

Also, you have to take into account that server could be updated, and JS not.